src/components/Form/Fields/File.js

```diff
@@ -43,23 +45,33 @@ class FileInput extends PureComponent {
     const {
       input: { value },
       accept,
+      label,
       className,
       completeClassName,
     } = this.props;
 
     return (
       <div className={cx(className, { [completeClassName]: !!value })}>
-        { value && this.props.children(value) }
-        <Dropzone
-          onDrop={this.onDrop}
-          multiple={false}
-          accept={accept}
-          className={cx('form-dropzone', { 'form-dropzone--replace': !!value })}
-          activeClassName="form-dropzone--active"
-          acceptClassName="form-dropzone--accept"
-          rejectClassName="form-dropzone--reject"
-          disabledClassName="form-dropzone--disabled"
-        />
+        { label &&
+          <div className="form-fields-file__label">{label}</div>
+        }
+        <div className="form-fields-file__dropzone">
+          { value &&
+            <div className="form-fields-file__dropzone-preview">
+              {this.props.children(value)}
+            </div>
+          }
+          <Dropzone
```

Comment:
Probably not part of this PR, but when I drop new images on the sociogram background, the previous one isn't removed from storage. Might be nice to delete, or highlight that there's an unused image taking up space. Let me know if I should file an issue.

Reply:
Good spot but we have an issue for this already!: https://github.com/codaco/Architect/issues/25